scripts/lib/mkcloud-driver-libvirt.sh

```diff
@@ -51,15 +51,20 @@ function libvirt_start_daemon()
 
 function libvirt_net_start()
 {
+    if (( $want_ipv6 > 0 )); then
+        $sudo sysctl -e net.ipv6.conf.all.accept_ra=2
```

Comment:
I feel this is dangerous -- we don't want to set it for all interfaces, especially on mkch* hosts. I think we should just deal if net-start fails, since the error from libvirt actually points out which interface requires accept_ra=2.